common/customizable_localization/00_knight_culture.txt

```diff
@@ -11,10 +11,6 @@
 		trigger = { wc_uses_knight_BloodKnight_trigger = yes }
```

Comment:
https://i.pinimg.com/originals/12/d6/ab/12d6ab9925f5119e39fa36c5b1df2439.png
https://preview.redd.it/umwtd9e9vm131.jpg?width=640&crop=smart&auto=webp&s=435b8ef11f55280c6ea6d605e836406f01c256af
https://imgur.com/EgPgr2M
Instead of...
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117541855-ef010780-b026-11eb-9c4b-9d373655b76b.png)

</details>
Because you can see taurens.